OpenRA.Platforms.Default/ThreadedGraphicsContext.cs

```diff
@@ -630,11 +637,19 @@ public void SetData(uint[,] colors)
 
 		public void SetData(byte[] colors, int width, int height)
 		{
-			// This creates some garbage for the GC to clean up,
-			// but allows us post a message instead of blocking the message queue by sending it.
-			var temp = new byte[colors.Length];
-			Array.Copy(colors, temp, temp.Length);
-			device.Post(setData2, (temp, width, height));
+			if (colors.Length < 85000)
```

Comment:
Can you add a comment here describing why this is 85000?

Reply:
Ok, I see that this is the size limit for putting things onto the LOH. It would still be good to have this in a comment!

Reply:
:+1: